com.raggesilver.BlackBox.json

```diff
@@ -0,0 +1,78 @@
+{
+  "app-id" : "com.raggesilver.BlackBox",
```

Comment:
the domain `raggesilver.com` doesn't exist, which mean the app-id doesn't meet the requirements.

This is also a problem in upstream code.